package.json

```diff
@@ -14,15 +14,17 @@
     "commitmsg": "validate-commit-msg"
   },
   "dependencies": {
-    "@dcl/schemas": "^6.9.0",
+    "@dcl/schemas": "^6.13.4",
     "@types/node": "^10.1.2",
+    "@walletconnect/ethereum-provider": "^2.5.2",
     "@walletconnect/web3-provider": "^1.6.5",
     "@web3-react/fortmatic-connector": "^6.1.6",
     "@web3-react/injected-connector": "^6.0.7",
     "@web3-react/network-connector": "^6.1.3",
     "@web3-react/types": "^6.0.7",
     "@web3-react/walletconnect-connector": "^7.0.2-alpha.0",
-    "@web3-react/walletlink-connector": "^6.2.13"
+    "@web3-react/walletlink-connector": "^6.2.13",
+    "@web3modal/standalone": "^2.2.2"
```

Comment:
Required to open the default wallet connect modal instead of a custom implementation